### ISSUE TITLE
React DevTools

### ISSUE THREAD

**Issue by KevinMolotov** [raphamorim/react-tv#137]
It looks like the react devtools aren't working for me. I'm always getting the message about React being in production mode even though I load the react-tv.development.js. Tried it on the example project built by React-TV and got the same problem. 

Any idea how to fix this?

**Comment by KevinMolotov** [comment]
Ok turns out we were missing one parameter in the injectIntoDevTools function of the renderer. Without 'version', it returns a production buildType instead of development.

I'll do a PR to fix this.

**Comment by raphamorim** [comment]
it's missing version key here: https://github.com/raphamorim/react-tv/blob/master/packages/react-tv/renderer/ReactTVFiberEntry.js#L274

**Comment by raphamorim** [comment]
It's failing even setting the version? @KevinMolotov

**Comment by KevinMolotov** [comment]
Hi @raphamorim! I added the missing version property but it only fixes the prod/dev issue where the devtools thought they were always in a production build. But we still get this message [https://i.imgur.com/GgUwenK.png](url).

I tried to switch the renderer with react-dom on the same project and the devtools works perfectly. Tried to update react-reconciler to check if the problem was coming from it, but there is too many breaking changes between the 0.10.0 and even the 0.11.0... And they're in 0.16.0 now.
Don't you have the same issue on your side?
